students/saleta_sebastian/lesson_02_flow_control/ladder.py

```diff
@@ -0,0 +1,14 @@
+from __future__ import print_function
+
+
+def print_ladder():
+    maxium_step = int(input())
```

Comment:
Typo in maxium_step. Did you mean maximum_step?

Reply:
yes, but now when I look at it this name is not clear.  I will change names of variables in this methods